fix minor typos (#44)
elong0527/r4csr: tlf-ae-summary.Rmd

# AE summary {#aesummary}

Following [ICH E3 guidance](https://database.ich.org/sites/default/files/E3_Guideline.pdf),

> we need the number of participants were included in each safety analysis in Section 12.2, Adverse Events (AEs).

Reviewer: ```suggestion
we summarize number of participants that were included in each safety analysis in Section 12.2, Adverse Events (AEs).
```
